insights/parsers/etc_libvirt_qemu_xml.py

```diff
@@ -0,0 +1,157 @@
+              <address type='pci' domain='0x0000' bus='0x00' slot='0x04' function='0x0'/>
+            </interface>
+            <interface type='hostdev' managed='yes'>
+              <mac address='b2:59:73:15:00:01'/>
+              <source>
+                <address type='pci' domain='0x0000' bus='0x04' slot='0x10' function='0x1'/>
+              </source>
+              <rom bar='on' file='/opt/vcp/share/ipxe/808610ed.rom'/>
+              <address type='pci' domain='0x0000' bus='0x00' slot='0x08' function='0x0'/>
+            </interface>
+            <serial type='pty'>
+              <target port='0'/>
+            </serial>
+            <console type='pty'>
+              <target type='serial' port='0'/>
+            </console>
+            <channel type='pipe'>
+              <source path='/var/lib/libvirt/qemu/channels/FROM-05-s00c06h0'/>
+              <target type='virtio' name='virtio2host'/>
+              <address type='virtio-serial' controller='0' bus='0' port='1'/>
+            </channel>
+            <channel type='pipe'>
+              <source path='/var/lib/libvirt/qemu/channels/HGC-05-s00c06h0'/>
+              <target type='virtio' name='virtio_host_guest_check'/>
+              <address type='virtio-serial' controller='0' bus='0' port='2'/>
+            </channel>
+            <input type='mouse' bus='ps2'/>
+            <input type='keyboard' bus='ps2'/>
+            <graphics type='vnc' port='-1' autoport='yes'>
+              <listen type='address'/>
+            </graphics>
+            <video>
+              <model type='cirrus' vram='16384' heads='1' primary='yes'/>
+              <address type='pci' domain='0x0000' bus='0x00' slot='0x02' function='0x0'/>
+            </video>
+            <watchdog model='i6300esb' action='reset'>
+              <address type='pci' domain='0x0000' bus='0x00' slot='0x07' function='0x0'/>
+            </watchdog>
+            <memballoon model='virtio'>
+              <address type='pci' domain='0x0000' bus='0x00' slot='0x06' function='0x0'/>
+            </memballoon>
+          </devices>
+        </domain>
+
+    Examples:
+        >>> xml_numa.file_name = 'vm.xml'
+        >>> assert xml_numa.vm_name == '05-s00c06h0'
+        >>> memnode = xml_numa.get_elements('./numatune/memnode', None)
+        >>> len(memnode[0].items()) == 3
+        True
+        >>> len(memnode[1].items()) == 3
+        True
+        >>> memnode[0].get('cellid') == '0'
+        True
+        >>> memnode[1].get('mode') == 'strict'
+        True
+    """
+    def parse_dom(self):
+        if self.dom is None:
+            return
+        else:
+            domain = {}
+            for child in self.dom:
+                if not child.getchildren():
+                    domain[child.tag] = child.text
+                else:
+                    domain[child.tag] = [c.items() for c in child.getchildren()]
+
+            return domain
+
+    @property
+    def vm_name(self):
+        if self.parse_dom():
```

Comment:
I believe you can just use `self.data` here since the base class `parse_content()` calls `parse_dom()` and saves the results in `self.data`.